scripts/superdesk/editor2/editor.js

```diff
@@ -1334,26 +1334,47 @@ angular.module('superdesk.editor2', [
         };
     }])
     .run(['embedService', 'iframelyService', function(embedService, iframelyService) {
-        var pattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
-        var loader = '//snappa.embed.pressassociation.io/playbuzz.js';
-        var html = [
+        var playBuzzPattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
+        var playBuzzlLoader = '//snappa.embed.pressassociation.io/playbuzz.js';
```

Comment:
I believe it does. This file is just a wrapper to support playbuzz embeds.  This can be useful for every instances